### DISCUSSION TITLE
Task / Workflow Execution persistence

### DISCUSSION THREAD

**Discussion by laxmikantb** [Netflix/conductor#2660] [Q&A]
Hi, 

I am using MySql as persistence layer , I do see task, workflow definition been persisted into MySql (meta-task_def, meta-workflow-def,queue, workflow, workflow-to-task tables ). but i am looking for status updates at each workflow and task which are completed or failed. Is this data should get persisted . is there any settings  needed for it. I can see status thru rest api but not is mysql.

nothing in task_in_progress, task_scheduled,workflow_pending 

Your help will be appreciated.

**Comment by aravindanr** [comment]
Workflow execution data includes its status and that information is persisted as well. In your case, the data should be in MySQL. You could also query it using `GET /workflow/{id}` api.
